autoptsclient_common.py

```diff
@@ -32,6 +32,7 @@
 import xml.etree.ElementTree as ET
 import time
 import datetime
+from termcolor import colored
```

Comment:
Note that using termcolor is not necessary, [btpclient.py](https://github.com/intel/auto-pts/blob/master/tools/btpclient.py#L44) is using SGR ANSI escape codes to set colors. Nevertheless, I do understand that termcolor makes things easier.

Reply:
I've tried using escape codes, at the beginning that seemed like the best way, but it turned out that using them messes up justifying text output in the terminal so I decided to use termcolor.